.github/workflows/stale.yml

```diff
@@ -20,9 +20,9 @@ jobs:
     - uses: actions/stale@v5
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-pr-message: 'This pull request has been automatically marked as stale because it has not had recent activity in the last 30 days. It will be closed in 3 days if no further activity occurs. Thank you for your contributions.'
+        stale-pr-message: 'This pull request has been automatically marked as stale because it has not had recent activity in the last 30 days. It will be closed in 7 days if no further activity occurs. Thank you for your contributions.'
         stale-pr-label: 'stale'
         days-before-pr-stale: 30
```

Comment:
We might bump up this to 100

Reply:
Roy says 35

Reply:
I'd be happier with 100 than 30. Minimize the amount of time wasted replying to a bot, I dont do PRs for fun, if it's becoming stale it s usally because the review process borked itself, not because I did not care

Reply:
I wouldn't mind pushing it to around 90-100 days. Why 35 though @GiudGiud @roystgnr ? You might have said in the meeting, but I think I missed that.

Reply:
Roy was commenting that the big IGA PR only stayed stale for at most 35 days in a row

Reply:
100 seems long to me but I don't care all that much. I'd prefer to make the whole team happy, so it sounds like 100 is the way to go